tests/test_error_handlers.py

```diff
@@ -211,22 +215,6 @@ def legacy_handler3(err, rq, rs, prms):
         client.simulate_get()
         client.simulate_head()
 
-    def test_handler_signature_shim_asgi(self):
```

Comment:
If someone is porting an old WSGI app over to ASGI, IMHO it is fair to require them to not use deprecated error handler signatures.